draft-ietf-rats-eat.md

```diff
@@ -1698,6 +1698,19 @@ an appropriate COSE hash algorithm, implying that an attacker cannot induce fals
 of a modified detached claims because the algorithms in the COSE registry are assumed to be
 of sufficient cryptographic strength.
 
+## Verification Keys {#verfication-key-sc}
+
+In all cases there must be some way that the verification key is itself verified or determined to be trustworthy.
+The key identification itself is never enough.
+This will always be by some out-of-band mechanism that is not described here.
+For example, the verifier may be configured with a root certificate or a master key by the verifier system administrator.
```

Comment:
```suggestion
For example, the verifier may be configured with a root certificate or a master key by a trusted entity such as the verifier system administrator.
```

Reply:
The word "trusted" is intentionally never used in the EAT document because to me it is too subjective. Trusted for what purpose and against what level of attack?